tests/integrations/import/util.rs

```diff
@@ -16,10 +16,7 @@ const CLEANUP_SST_MILLIS: u64 = 10;
 pub fn new_cluster(cfg: TikvConfig) -> (Cluster<RocksEngine, ServerCluster<RocksEngine>>, Context) {
     let count = 1;
     let mut cluster = new_server_cluster(0, count);
-    cluster.cfg = Config {
-        tikv: cfg,
-        prefer_mem: true,
-    };
+    cluster.set_cfg(cfg);
```

Comment:
Why not directly use `Config::new(cfg, true)` ?

Reply:
Just to reuse the current config's temp dir and avoid removing&creating a new temp dir.